src/features/commands/character/character-command.service.ts

```diff
@@ -17,7 +17,7 @@ export class CharacterCommandService {
   public message(message: Message): void {
     CharacterService.getInstance().getEntity(message.author.id)
     .then((characterFound) => {
-      const boldCharacterName = MessageFormattingBoldererService.getInstance().bold(characterFound.name)
+      const boldCharacterName = MessageFormattingService.getInstance().format('bold', characterFound.name)
```

Comment:
Use an enum instead of a string for 'bold'.

Reply:
I see. I will use my enum